config/manager/manager.yaml

```diff
@@ -1,10 +1,3 @@
-apiVersion: v1
```

Comment:
why remove this?

Reply:
I found it difficult to debug with edgex-system, so I just used the default namespace🤣.

Reply:
Now we have `helm`, and we can deploy to other namespaces use it. I don't keep the kustomize namespace

Reply:
Sorry, I find this deletion unnecessary. Let me modify that🤣

Reply:
If my namespace is default, every time `make undeploy` will show that the namespace cannot be deleted, so I deleted that for now, and I don't know if there's any other way to avoid this problem. I definitely use the default namespace most of the time when I'm developing.